dtfs-central-api/src/services/state-machines/fee-record/fee-record.state-machine.test.ts

```diff
@@ -75,6 +77,47 @@ describe('FeeRecordStateMachine', () => {
       // Assert
       expect(handleFeeRecordPaymentAddedEvent).toHaveBeenCalledTimes(1);
     });
+
+    it(`handles the '${FEE_RECORD_EVENT_TYPE.CORRECTION_REQUESTED}' event`, async () => {
+      // Arrange
+      const stateMachine = FeeRecordStateMachine.forFeeRecord(TO_DO_FEE_RECORD);
+
+      // Act
+      await stateMachine.handleEvent({
+        type: FEE_RECORD_EVENT_TYPE.CORRECTION_REQUESTED,
+        payload: {
+          transactionEntityManager: {} as unknown as EntityManager,
```

Comment:
do we need as unknown here?

Reply:
I had copied it from the other tests, assumed it broke an eslint rule if not, but actually seems to be ok, will change!